pkg/schedule/schedule.go

```diff
@@ -148,6 +148,88 @@ func checkTrigger(
 	return false, nil
 }
 
+func hasTriggered(
+	lastTrigger time.Time,
+	lastTriggerEarliest time.Time,
+	now time.Time,
+) (bool, error) {
+	// If the last trigger is before the  we had triggered before the scheduled time this month, don't
+	// triggered again
```

Comment:
take another look at this comment and correct it.

Reply:
Modified.